extensions/mssql/config.json

```diff
@@ -1,6 +1,6 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/v{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
-	"version": "3.0.0-release.16",
+	"version": "3.0.0-release.22",
```

Comment:
Is this version bump necessary for this PR?

Reply:
not exactly for this PR. This version contains some fixes for the extenstion, so I'd like to update the version of SQL tools service. I'm not sure how is it usually happens.

Reply:
All good, just wanted to make sure it was intended.